src/projectContent.js

```diff
@@ -0,0 +1,25 @@
+/* global document */
+
+const projectContentDom = () => {
```

Comment:
The stuff contained here are static. We don't need javascript to render the html tags for these. I suggest we just code them directly into `index.html`. The same goes for the stuff in `todoContentDom.js` and `navBar.js`